install/kubernetes/helm/meshery/charts/mesheryconsul/values.yaml

```diff
@@ -0,0 +1,90 @@
+# Default values for mesheryistio.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: layer5/meshery-consul:latest
```

Comment:
@leecalcote should this be `stable-latest`? It is in .meshery/meshery.yaml

Reply:
Good catch. Yes, this tag needs to be stable-latest. I suspect this update is needed across all other charts, too.